etc/kayobe/kolla/config/prometheus/prometheus.yml.d/60-redfish.yml

```
@@ -0,0 +1,27 @@
+# yamllint disable-file
+---
+{% if seed_redfish_exporter_container_enabled | bool %}
+scrape_configs:
+  - job_name: redfish-exporter-seed
+    metrics_path: /redfish
+    scrape_timeout: 120s
+    scrape_interval: {{ 8 * groups['redfish_exporter_targets'] | length }}s
```

If the size of the group `redfish_exporter_targets` is too small then the interval will be smaller than the timeout. 

```suggestion
    scrape_timeout: 60s
    scrape_interval: {{ [8 * groups['redfish_exporter_targets'] | length, 60] | min }}s
```

---

Might be better to keep the timeout of 120s and apply a minimum of 120s?

---

Okay. Should it be capped at 120s or should it increase as the group grows larger. So replace `min` fitler with `max`

---

and need to use the max filter

---

Done.